### ISSUE TITLE
Hide Password

### ISSUE THREAD

**Issue by ahmadnassri** [ahmadnassri/app-restconsole#5]
I would like to see the password field under the Autorization tab as a real password field and not plain text.

**Comment by phpwalter** [comment]
I don't agree. This is a development phase tool. I should not be using sensitive passwords. If others insist on PASSWORD (astrick) type display, than may I suggest a check box that allows me to turn on and off this "feature".

I find PASSWORD fields not easy to work with. Can't tell you how many times that a fat fingered password, entered into a PASSWORD type field, has proved to be the problem I've been tracking for hours at a time.

**Comment by ahmadnassri** [comment]
That's how I was going to implement it actually, it should be an option, not one way or another.

**Comment by phpwalter** [comment]
great, thank you
